### PR TITLE
Exams by Nazarii Radichenko IM-22

### PR DESCRIPTION
**Pull request by radichenko**

### CODE REVIEW

**Comment by KobbAsa** [comment]
Looks good to me. Code became more readable for sure.

File: Tasks/month.js [hunk 1]
```diff
@@ -15,12 +17,13 @@ Months = [
   'dec',
 ];
 
-Month = (s) => {
-  l = Months.length;
-  for (let i = 0; i < l; i++) {
-    if (s.toLowerCase().startsWith(Months[i])) return i + 1;
+const getMonthNumber = (month) => {
```
Comment (stasyan04) [inline Tasks/month.js:20]:
```suggestion
const getMonthNumber = month => {
```
As for me, it looks better without brackets (  ), but both are good!

File: Tasks/types.js [hunk 2]
```diff
@@ -1,18 +1,18 @@
 // Count types in an array
 
-types_ = function (s) {
-  types_ = {
+'use strict';
+
+const countTypes = (array) => {
```
Comment (stasyan04) [inline Tasks/types.js:5]:
```suggestion
const countTypes = array => {
```
Here you can also do it without brackets (  ), but it`s your choice.

**Review by stasyan04 — COMMENTED** [review]
Your code is good!

**Comment by M-Rybalko** [comment]
LGTM. Good job!

File: .eslintrc.json [hunk 3]
```diff
@@ -18,7 +18,7 @@
     ],
     "linebreak-style": [
       "error",
-      "unix"
+      "windows"
```
Comment (tshemsedinov) [inline .eslintrc.json:21]: https://youtu.be/W87wOCSPA08

File: Tasks/month.js (hunk 1 above)
Comment (tshemsedinov) [reply Tasks/month.js:20]: We have common eslint config for that

**Review by tshemsedinov — APPROVED** [review]
LGTM (looks good to me)

**Pull request closed without merging**